mergify_engine/queue.py

```diff
@@ -118,16 +169,39 @@ def add_pull(self, ctxt: context.Context, config: QueueConfig) -> None:
         )
 
         if added:
+            if self.train:
+                # NOTE(sileht): We don't care about checking the queue name here, but we
+                # please mypy
```

Comment:
maybe go with:
```suggestion
            if not self.train:
                # NOTE(sileht): We don't care about checking the queue name here, but we
                # please mypy
                raise RuntimeError(…)
```
rather than failing silently?
If we ever modify this code later we might have a bug and miss it.

Reply:
The comment is wrong, it's a leftover of another `if` and I changed the code upstream (L156)  to avoid the `None` checking everywhere to please just mypy.